src/organizations/components/OrgProfileTab/LeaveOrg.tsx

```diff
@@ -40,18 +45,15 @@ export const LeaveOrgButton: FC = () => {
         <p className="org-profile-tab--heading org-profile-tab--deleteHeading">
           Leave the <b>{org.name}</b> organization.
         </p>
-        <ConfirmationButton
+        <Button
           className="org-profile-tab--leaveOrgButton"
           color={ComponentColor.Default}
-          confirmationButtonColor={ComponentColor.Danger}
```

Comment:
Can't use a `ConfirmationButton` here since the figma calls for a full-on overlay to warn the user and provide a dropdown for identifying which user will receive the idpe assets.